2주차/signin.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>로그인</title>
+    <link rel="stylesheet" href="./src/reset.css" />
+    <link rel="stylesheet" href="./src/style.css" />
+  </head>
+  <body style="background-color: #F0F6FF;">
+    <div class="loginbox">
+      <a href="index.html">
+        <img src="./images/logo.svg", width="210.583px", height="38px">
+      </a>
+      <div class="askMember">
+        회원이 아니신가요? <a href="signup.html" class="gosignup">회원가입하기</a>
+      </div>
+      <div class="signininputbox">
```

Comment:
```suggestion
      <div class="sign-in-input-box">
```
위와같은 클래스네임은 주로 짓는 국룰 방법이 있는데 관련 링크 첨부할게요! https://velog.io/@leyuri/%ED%91%9C%EA%B8%B0%EB%B2%95-%EC%8A%A4%EB%84%A4%EC%9D%B4%ED%81%AC-%EC%BC%80%EC%9D%B4%EC%8A%A4-%ED%8C%8C%EC%8A%A4%EC%B9%BC-%EC%BC%80%EC%9D%B4%EC%8A%A4-%EC%B9%B4%EB%A9%9C-%EC%BC%80%EC%9D%B4%EC%8A%A4